linchpin/api/__init__.py

```diff
@@ -145,6 +142,7 @@ def run_playbook(self, pinfile, targets='all', playbook='up'):
                                                 console=self.console)
```

Comment:
Should it not have a dictionary of results in the above block also?  Looks like this could also have multiple targets also. I would think length of 0 below is for all. This block is for 1 or more.